### PR TITLE
Some typos and a missing global

### PR DESCRIPTION
**Pull request by bic-ed**
Spotted using a debugging tool. Some of this are very likely producing a bug, tough I didn't test each one of them. I'm opening this as a draft because if you wish I'll try to address some more errors I found.

### CODE REVIEW

**Comment by acrylian** [comment]
Thanks, looks like good finds!

File: zp-core/zp-extensions/user-expiry.php [hunk 1]
```diff
@@ -250,6 +250,7 @@ static function checklogon($loggedin, $user) {
 	 * @return string
 	 */
 	static function reverify($path) {
+		global $_zp_current_admin_obj;
 		//process any verifications posted
 		if (isset($_GET['user_expiry_reverify'])) {
 			$params = sanitize(unserialize(pack("H*", trim($_GET['user_expiry_reverify']), '.'), ['allowed_classes' => false]));
```
Comment (acrylian) [inline zp-core/zp-extensions/user-expiry.php:256]: This seems unnecessary as I cannot see the global being used within this method. But it does not hurt…

Comment (bic-ed) [reply zp-core/zp-extensions/user-expiry.php:256]: Here
[268: $userobj->setLastChangeUser($_zp_current_admin_obj->getUser());](https://github.com/bic-ed/zenphoto/blob/813293227b2e68e07ceabfa44970447c794c5aa6/zp-core/zp-extensions/user-expiry.php#L268)

Comment (acrylian) [reply zp-core/zp-extensions/user-expiry.php:256]: Ah, sorry, you are totally right… (looked at another release stream where this was missing)

**Comment by bic-ed** [comment]
Hi, I've just pushed on my fork two more small typo fixes, along with the updated (again) Italian translation. Before opening a new pull request here, I'd like to be sure you have the time to review changes and the will to merge them before finally releasing v1.6.
See bb4a86faf813c2cbb998f302a7ea5c1451f8c482

**Pull request merged**